src/SignalRServiceExtension/Config/StaticServiceHubContextStore.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
     /// A global <see cref="IServiceManagerStore"/> for the extension.
     /// It stores <see cref="IServiceHubContextStore"/> per connection string.
     /// </summary>
-    public static class StaticServiceHubContextStore
```

Comment:
don't make it internal, it can be used by C# users. check out [https://github.com/Azure/azure-functions-signalrservice-extension/blob/dev/samples/simple-chat/csharp/FunctionApp/Functions.cs#L42](https://github.com/Azure/azure-functions-signalrservice-extension/blob/dev/samples/simple-chat/csharp/FunctionApp/Functions.cs#L42)

Reply:
Why C# users want to use `ServiceManager`?

Reply:
in line 47, add/modify/... their custom claims.

Reply:
Could we add claims as a property of `SignalRConnectionInfo` ? Then other languages are able to custom claims too.

Reply:
In addition, `IServiceContext` will replace `IServiceManager` in the future and there will be breaking changes anyway.

Reply:
What if cx need some pre-process before generating the token?
e.g. the header `A` is 1.But cx wants claim type = A*3, value = 1 * 123. then the claim is ("AAA", 123)

Reply:
Why do cx want to do this? No one using other languages requests such feature, I think they are able to do this in other ways.

Reply:
How can you predict cx's usage?

Reply:
Also cx can send messages via this, especially for those familiar signalr library